src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -29,13 +29,11 @@ interface BidResultProps {
 
 const messageForPollingTimeout = {
   title: "Bid processing",
-  description: `
-    We’re receiving a high volume of traffic
-    and your bid is still processing.
-
-    If you don’t receive an update soon,
-    please contact [support@artsy.net](mailto:support@artsy.net).
-  `,
+  description:
+    "We’re receiving a high volume of traffic\n" +
```

Comment:
There's no way to just have a string of text that we wrap due to the size of the screen, right? 😬

Reply:
It's easier to use the `\n` to add line breaks rather than relying on the screen size. Though It is still possible that on iPhone SE this ends up with an unexpected line break.

Reply:
Actually I didn't mean to include this change as part of this PR. I'll update to remove the commit.